test/XRP/helpers/xrp-test-utils.ts

```diff
@@ -0,0 +1,410 @@
+import Utils from '../../../src/Common/utils'
+import { AccountAddress } from '../../../src/XRP/generated/org/xrpl/rpc/v1/account_pb'
+import {
+  CurrencyAmount,
+  XRPDropsAmount,
+  Currency,
+  IssuedCurrencyAmount,
+} from '../../../src/XRP/generated/org/xrpl/rpc/v1/amount_pb'
+import {
+  Account,
+  Amount,
+  Destination,
+  Domain,
+  Sequence,
+  SigningPublicKey,
+  Authorize,
+  Unauthorize,
+  ClearFlag,
+  EmailHash,
+  MessageKey,
+  SetFlag,
+  TransferRate,
+  TickSize,
+} from '../../../src/XRP/generated/org/xrpl/rpc/v1/common_pb'
+import {
+  Payment,
+  Transaction,
+  DepositPreauth,
+  AccountSet,
+} from '../../../src/XRP/generated/org/xrpl/rpc/v1/transaction_pb'
+
+const xrpTestUtils = {
```

Comment:
this seems fine but in ES6, native modules are objects key'd by the symbol they're exporting. there are a few ways you can do this:

```
function a() {}
function b() {}

export default { a, b }
```

since the importer names the import, this is equivalent to what you have right now and then doing `import xrpTestUtils from 'pathtothisfile'`. the downside of this is that you must explicitly add it to the exported object, but i feel it's a bit easier to see what things are being exported by name

the default export is always named by the exporter (you can think of it as a doing a `let xrpTestUtils = module.default` thus, you could do something `import myBoguslyNamedFunction from 'pathtothisfile`

if you want, i think it's also fine to just not have a default export. if you remove the default export but export all of the functions in this file, at the import site you can do

```
import * as xrpTestUtils from 'pathtothisfile'
```

the second is my preference but the sdk seems to prefer explicit namespaced exports. it lets you import only the functions you want but you lose the explicit `xrpTestUtils` token. up to you

Reply:
This makes sense for the most part.  For now, and since it's test code, I'm going to leave it in the same style as the other modules in `x-c-js` - but in future will favor the latter.